scripts/prepdocs/strategy.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC
```

Comment:
This could use module docstring and class docstrings, particularly for Strategy since thats a generic name, not clear how it's to be used.

Reply:
good point